catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/impl/ResourceCacheImplTest.java

```diff
@@ -117,6 +129,11 @@ public void setUp() throws MalformedURLException, URISyntaxException {
     public void teardownTest() {
         try {
             FileUtils.cleanDirectory(new File(defaultProductCacheDirectory));
+            File windowsTempFileNeedsCleaning = new File(
+                    workingDir + File.separator + "dataProduct_Cache");
+            if (windowsTempFileNeedsCleaning.exists()) {
+                FileUtils.cleanDirectory(windowsTempFileNeedsCleaning);
+            }
```

Comment:
The above code block is a bit of an anomaly. Since this test class will likely go away after the rewrite in-progress for the download system, this fix will get us through on Windows until that time. Not sure it's worth it to track down where the separator `\` is getting omitted. That's strange behavior even for Windows. 

On Mac/UNIX, the directory `dataProduct_Cache` is never created.